extensions/kube-object-event-status/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kube-object-event-status",
-  "version": "0.1.0",
+  "version": "0.0.1",
```

Comment:
Marked these as `0.0.1` to indicate more clearly that this version does not matter.

Reply:
Wouldn't that be `0.0.0` then? Like the NPM package?